docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md

```diff
@@ -123,6 +123,41 @@ Enter prompts into the textbox at the bottom and press the **Generate** button t
 To shut down the WebUI server, use **Ctrl+C** in the **Anaconda Prompt** terminal where the WebUI Server is runing, then close your browser tab.
 
 
+## 5. Advanced Usage
+### Using Instruct Chat Mode
+Instruction-following models are models that are fine-tuned with specific prompt formats. 
+For these models, you should ideally use the `instruct` chat mode.
+Under this mode, the model receives user prompts that are formatted according to prompt formats it was trained with.
+
+To use `instruct` chat mode, select `chat` tab, scroll down the page, and then select `instruct` under `Mode`.
+
+<a href="https://llm-assets.readthedocs.io/en/latest/_images/webui_quickstart_chat_mode_instruct.png">
```

Comment:
can we make it open the original image in a new window? maybe more friendly for users?

Reply:
This is already done for all images in the quickstart, including current images.

Reply:
I mean in a new window, currently it goes in the same window, users need to click the back bottom to view the doc page again, seems not convenient?